src/control-panel/control-panel.js

```diff
@@ -1,51 +1,50 @@
-const {getReadlineInterface, clearScreen} = require('../console-helper/utils');
-
-
-const controlPanel = () => {
-
-    let row = null;
-    let column = null;
-
-    const readlineInterface = getReadlineInterface();
-
-    let takeInputForRow = (player) => {
-        readlineInterface.question(`Player 1 Turn - ${player} Enter Row (1, 2, 3):- `, (givenRow) => {
-
-            if (givenRow === '1' || givenRow === '2' || givenRow === '3') {
-                row = parseInt(givenRow) - 1;
-
-                console.log('message', givenRow);
-                console.log('message', row);
-
-                takeInputForColumn(player);
-
-            } else {
-                takeInputForRow(player);
-            }
-        });
-    };
+const {
+    getReadlineInterface,
+    clearConsole,
+    getPlayer1,
+    getPlayer2,
+    getPlayerNo
+} = require('../console-helper/utils');
+
+let row = null;
+let column = null;
+const readlineInterface = getReadlineInterface();
+
+let takeInputForRow = (player) => {
+    let playerNumber = getPlayerNo(player);
+    readlineInterface.question(`[Player ${playerNumber} = ${player}] - Enter Row [1, 2, 3] :- `, (givenRow) => {
+        if (givenRow === '1' || givenRow === '2' || givenRow === '3') {
+            row = parseInt(givenRow) - 1;
+            takeInputForColumn(player);
+        } else {
+            takeInputForRow(player);
+        }
+    });
+};
 
-    const takeInputForColumn = (player) => {
-        readlineInterface.question(`Player 2 Turn - ${player} Enter Column (1, 2, 3):- `, (givenColumn) => {
+const takeInputForColumn = (player) => {
+    let playerNumber = getPlayerNo(player);
+    readlineInterface.question(`[Player ${playerNumber} = ${player}] - Enter Column [1, 2, 3] :- `, (givenColumn) => {
+        if (givenColumn === '1' || givenColumn === '2' || givenColumn === '3') {
+            column = parseInt(givenColumn) - 1;
 
-            if (givenColumn === '0' || givenColumn === '1' || givenColumn === '2') {
-                column = parseInt(givenColumn) - 1;
+            //todo: need to process row, column and player information
```

Comment:
Task Plan We can do.
- base row, column, and player input put X or O symbol in the board array
- draw your game board with it's selected input
- then also check which user is going to win

I think it will be work.